### PR TITLE
Rewrite according to updated concepts

### PR DESCRIPTION
**Pull request by Oshratn**
Based on the discussion on this external doc: https://www.dropbox.com/scl/fi/5yoe04g2dzc5y5q2laj9h/App-Catalog-on-docs.giantswarm.paper?dl=0&rlkey=m3527f2kw7wibdhzyr2to1bs6

### CODE REVIEW

File: src/content/basics/app-catalog/index.md [hunk 1]
```diff
@@ -1,97 +1,97 @@
...
 
-The _Giant Swarm App Catalog_ refers to a set of features and concepts that allow
-you to browse, install and manage the configurations of apps (such as prometheus)
+The _Giant Swarm App Platform_ refers to a set of features and concepts that allow
+you to browse, install and manage the configurations of apps (such as Prometheus)
 from a single place; the Control Plane.
 
-Through this we are also providing you a list of curated _Managed Apps_.
-In the near future you might also see _Apps_ provided by our partners.
+Using this platform we will provide a collection of curated _Apps_. These _Apps_ are grouped into _App Catalogs_, which is browesable through our web interface.
```
Comment (puja108) [inline src/content/basics/app-catalog/index.md:17]:
```suggestion
Using this platform we will provide a collection of curated _Apps_. These _Apps_ are grouped into _App Catalogs_, which are browsable through our web interface.
```

**Comment by Oshratn** [comment]
linkcheck failed. What does this mean? How do I fix it?

**Comment by oponder** [comment]
@Oshratn I'm fixing the link checker! sorry for any confusion in that

**Comment by oponder** [comment]
It's beyond my control for now, related to other websites not working ,possibly due to github pages having issues right now. 

So don't worry if you're seeing linkchecker failures for now.

I think we should probably consider removing linkchecker from the CI, and instead make it some kind of periodic action that creates issues for us or pings us on slack when dead links are found. 

(fyi @marians, would you agree that we should remove linkchecker from the CI, given that we can find some other solution to staying aware of broken links?)

**Comment by Oshratn** [comment]
Waiting on @cokiengchiara to green-light for when we are live with the cleaned-up catalog.

**Comment by cokiengchiara** [comment]
> Waiting on @cokiengchiara to green-light for when we are live with the cleaned-up catalog.

Almost there cc @glitchcrab https://gigantic.slack.com/archives/CR99LSZ1N/p1586350073110100

**Comment by glitchcrab** [comment]
> Almost there cc @glitchcrab https://gigantic.slack.com/archives/CR99LSZ1N/p1586350073110100

it's dead

chaosmonkey

File: src/content/basics/app-catalog/index.md [hunk 2]
```diff
@@ -1,97 +1,97 @@
...
-Custom Resources, some created by us, and others created by you, which make up
-the desired state of the Giant Swarm App Catalog.
+Custom Resources, some created by us, and others created by you. Together, they make up
+the desired state of the App Platform.
 
-For example, this "App" Custom Resource indicates that you want Instana installed
+For example, this "App" Custom Resource indicates that you want Kong installed
 on a specific tenant cluster.
 
 ```
```
Comment (marians) [inline src/content/basics/app-catalog/index.md:33]:
```suggestion
```yaml
```

Comment (marians) [reply src/content/basics/app-catalog/index.md:33]: For proper syntax highlighting

**Comment by Oshratn** [comment]
This conflicts with #440 @teemow / @puja108 help please.

**Comment by teemow** [comment]
@Oshratn should be good now

**Pull request merged**